tests/test_position_transfers.py

```diff
@@ -0,0 +1,601 @@
+    assert result.transaction_id
+
+
+@pytest.mark.parametrize(
+    "from_subaccount,to_subaccount,global_direction",
+    [
+        (SM_SUBACCOUNT_ID, PM_SUBACCOUNT_ID, "buy"),
+        (PM_SUBACCOUNT_ID, SM_SUBACCOUNT_ID, "sell"),
+        (SM_SUBACCOUNT_ID, PM_SUBACCOUNT_ID, "sell"),
+        (PM_SUBACCOUNT_ID, SM_SUBACCOUNT_ID, "buy"),
+    ],
+)
+def test_transfer_positions_between_subaccount_types(derive_client, from_subaccount, to_subaccount, global_direction):
+    """Test transfer_positions between different subaccount types."""
+    positions = [
+        TransferPosition(
+            instrument_name="ETH-PERP",
+            amount=0.1,
+            limit_price=2500.0,
+        ),
+        TransferPosition(
+            instrument_name="BTC-PERP",
+            amount=0.01,
+            limit_price=45000.0,
+        ),
+    ]
+
+    result = derive_client.transfer_positions(
+        positions=positions,
+        from_subaccount_id=from_subaccount,
+        to_subaccount_id=to_subaccount,
+        global_direction=global_direction,
+    )
+
+    assert result is not None
+    assert result.transaction_id
+    assert result.status
+
+
+def test_transfer_positions_multiple_instruments(derive_client):
+    """Test transfer_positions with multiple different instruments."""
+    # Get available subaccounts
+    subaccounts = derive_client.fetch_subaccounts()
+    subaccount_ids = subaccounts['subaccount_ids']
+
+    if len(subaccount_ids) < 2:
+        pytest.skip("Need at least 2 subaccounts for position transfer test")
+
+    from_subaccount_id = subaccount_ids[0]
+    to_subaccount_id = subaccount_ids[1]
+
+    # Get available instruments to ensure we test with valid ones
+    perp_instruments = derive_client.fetch_instruments(instrument_type=InstrumentType.PERP)
+
+    if len(perp_instruments) < 3:
+        pytest.skip("Need at least 3 perpetual instruments for comprehensive test")
+
+    # Use first 3 available instruments
+    positions = []
+    for i, instrument in enumerate(perp_instruments[:3]):
+        positions.append(
+            TransferPosition(
+                instrument_name=instrument["instrument_name"],
+                amount=0.1 * (i + 1),  # Varying amounts
+                limit_price=1000.0 + (i * 1000),  # Varying prices
+            )
+        )
+
+    result = derive_client.transfer_positions(
+        positions=positions,
+        from_subaccount_id=from_subaccount_id,
+        to_subaccount_id=to_subaccount_id,
+        global_direction="buy",
+    )
+
+    assert result is not None
+    assert result.transaction_id
+
+
+@pytest.mark.parametrize(
+    "amount",
+    TRANSFER_AMOUNTS,
+)
+def test_transfer_position_different_amounts(derive_client, amount):
+    """Test transfer_position with different transfer amounts."""
+    # Get available subaccounts
+    subaccounts = derive_client.fetch_subaccounts()
+    subaccount_ids = subaccounts['subaccount_ids']
+
+    if len(subaccount_ids) < 2:
```

Comment:
the subaccount used for testing definitely has > 2 subaccounts
This should be removed on all tests